source/opt/local_single_block_elim_pass.cpp

```diff
@@ -202,6 +202,14 @@ void LocalSingleBlockLoadStoreElimPass::DCEInst(ir::Instruction* inst) {
 
 bool LocalSingleBlockLoadStoreElimPass::LocalSingleBlockLoadStoreElim(
     ir::Function* func) {
+  // Verify no CopyObject ops in function. This is a pre-SSA pass and
+  // is generally is not useful for code already in CSSA form.
```

Comment:
Good comment.
Personally I'd spell out "Conventional SSA" since it's not universally known.    (As in, I didn't know it and found the wrong thing first time I looked.) But this is ok.